data/korath/korath missions.txt

```diff
@@ -257,3 +257,54 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+				"reputation: Korath" >= -100
+			and
+				random < 10
+				"reputation: Korath" < -100
+				"reputation: Korath" >= -1000
+			and
+				random < 25
+				"reputation: Korath" < -1000
+				"reputation: Korath" >= -10000
+			and
+				random < 50
+				"reputation: Korath" < -10000
+
+			and
+				random < 10
+				"reputation: Korath (Civilian)" < 0
+				"reputation: Korath (Civilian)" >= -300
+			and
+				random < 25
+				"reputation: Korath (Civilian)" < -300
+				"reputation: Korath (Civilian)" >= -3000
+			and
+				random < 50
+				"reputation: Korath (Civilian)" < -3000
+	source
+		government Korath "Korath (Civilian)"
+		system Alcyone Almach "Alpha Hydri" Atik Durax Misam Polaris Sheratan
+	on offer
+		"korath: boarding response count" ++
+		conversation
+			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. The pulse throws your ship off theirs, but it appears you could board them again. You had best finish your business here before you are reduced to a ball of superheated plasma."
```

Comment:
```suggestion
			"An angry hiss comes over your radio, and you see a strange pulse radiate from the hull of the Korath vessel. The pulse throws your ship off theirs, but it appears you could board them again. Suddenly, your ship detects a powerful hyperwave emission followed by several Korath ships jumping in. You had best finish your business here before you are reduced to a ball of superheated plasma."
```
I think this needs to be adjusted a bit. "...pulse along" doesn't really make sense as much as a form of "....pulse from". We also don't really refer to ships as "Capital ships" (apart from the Remnant license), and should probably leave that vague and reference what we do know about the ships being Korath. 

Reorganized wording to make a bit more sense/flow better. If the pulse is the hyperwave emission itself, and not the JD's of other ships, the wording could be connected a bit better.

Reply:
@roadrunner56 Thoughts as well?

Reply:
I already merged this change, but roadrunner56 has not indicated the presence or lack of a thought.

Reply:
Sorry, I've been kind of busy over the past few months and I must have missed this. Saugia's change looks good, if that's still an issue.